client/analysisui/iconsupport.css

```diff
@@ -0,0 +1,40 @@
+/*textbox sizing*/
+.jmv-option-icon[data-icon=ContinuousVariable] {
```

Comment:
I can do that but the ContinuousVariable is a yaml enum as well. The only thought i had was to make the css attr all lowercase as a rule so use ContinuousVariable in the yaml and continuousvariable in the css.